modules/040-node-manager/hooks/discover_standby_ng.go

```diff
@@ -63,6 +63,8 @@ func standbyNodeGroupFilter(obj *unstructured.Unstructured) (go_hook.FilterResul
 
 	needStandby := false
 	maxPerZone := 0
+	overprovisioningRate := int64(50) // default: 50%
```

Comment:
You have default in openapi spec, why you set default in this place ?

Reply:
standbyHolder section could not be set. User can set only `standby: 5` for example. Sure we can put `default: {}` there but it will generate extra values, which are usually unneeded.
We should have struct like this:
```
standby:
  type: object
  properties:
    count:
      type: integer
    overprovisioningRate:
      type: integer
```
honestly but not now.
Maybe I have missed... and I have to remove default value from an openapi spec